sdk/nodejs/storage/zMixins.ts

```diff
@@ -285,6 +311,12 @@ export type QueueEventSubscriptionArgs = util.Overwrite<appservice.CallbackFunct
      * used.
      */
     location?: pulumi.Input<string>;
+
+    /** 
+     * Host settings specific to the Storage Queue plugin. These values can be provided here, or defaults will 
+     * be used in their place. 
+     */
+    hostSettings?: QueueHostSettings;
 }>;

```

Comment:
can we extract teh topic/queue stuff out into separate PRs?

Reply:
We can... why?

Reply:
i just preferred keeping the commits nice and distinct :)  it's not a huge deal though!

Reply:
This is cohesive enough IMO. I'll do so next time!